### ISSUE TITLE
Add custom virtual background images to Greenlight.

### ISSUE THREAD

**Issue by eduardsl** [bigbluebutton/greenlight#3004]
![1](https://user-images.githubusercontent.com/91247506/145402292-7caea96e-f847-464e-a56d-45724f853acb.png)
Hello! I'm fairly new to Greenlight and BBB in general. I currently have a docker instalation of Greenlight for which I've customized the landing page, mail templates etc. by adding the necessary volumes to the docker-compose.yml and customizing them according to the documentation (https://docs.bigbluebutton.org/greenlight/gl-customize.html).

The sistem works great, but I would like to add more options to the virtual web-cam backgrounds. Currently I'm able to select "Blur" or one of 3 preset images - "Home", "Coffeeshop" or "Board". I would like to add some organization specific backgrounds, but I can't seem to find where these are defined or where the images are stored. I tried grep-ing the container for the corresponding background names with no luck. 

TL;DR - How do I add custom virtual background images to dockerized Greenlight install?

Thanks! :)

-Eduards

**Comment by eduardsl** [comment]
My bad, I realized this is configurable for BBB in general, not just Greenlight. (https://docs.bigbluebutton.org/admin/customize.html#use-custom-images-for-virtual-background)
